.github/workflows/scripts/scanner-fetch-nvd.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -euo pipefail
+
+base_url="https://services.nvd.nist.gov/rest/json/cves/2.0"
+dir="nvd-data"
+mkdir -p "$dir"
+
+info() { echo >&2 "$(date): INFO:" "$@"; }
+
+# Compression and upload
+compress_upload() {
+  tar -czf "$dir.tar.gz" -C "$dir" .
+  info "Compressed file details: $(ls -lh "$dir.tar.gz")"
+  mkdir -p "nvd-bundle" && mv "$dir.tar.gz" "nvd-bundle/$dir.tar.gz"
+  gsutil cp -r "nvd-bundle" "gs://scanner-v4-test/" || { info "Upload failed"; exit 1; }
```

Comment:
Have we decided on a final bucket to store this data? If so, any reason not to use it now? If not, please create a comment & ticket for ensuring we update to the proper bucket in the future.